### DISCUSSION TITLE
我该怎么办

### DISCUSSION THREAD

**Discussion by dhicoc** [tw93/Pake#556] [Q&A] [answered]
C:\Users\DELL>pake https://www.doubao.com/ --name DouBao --icon "F:\Google下载\1038679_1689570098452157843.png~tplv-a9rns2rl98-image-qvalue.ico"
✼ The first use requires installing system dependencies.
✼ See more in https://tauri.app/v1/guides/getting-started/prerequisites.
✸ Installing package...

✹ Installing package...

68 packages are looking for funding
√ Package installed!
✼ The first packaging may be slow, please be patient and wait, it will be faster afterwards.
✺ Building app...

> pake-cli@2.3.1 build
> npm run tauri build --release
✹ Building app...

> pake-cli@2.3.1 tauri
> tauri build
    Updating `rsproxy` index
fatal: detected dubious ownership in repository at 'C:/Users/DELL/.cargo/registry/index/rsproxy.cn-8f6827c7555bfaf8/.git'
'C:/Users/DELL/.cargo/registry/index/rsproxy.cn-8f6827c7555bfaf8/.git' is owned by:
        'S-1-5-32-544'
but the current user is:
        'S-1-5-21-394297639-2468674266-1266804290-1001'
To add an exception for this directory, call:

        git config --global --add safe.directory C:/Users/DELL/.cargo/registry/index/rsproxy.cn-8f6827c7555bfaf8/.git
error: failed to get `tauri-plugin-window-state` as a dependency of package `app v0.1.0 (C:\Users\DELL\AppData\Roaming\npm\node_modules\pake-cli\src-tauri)`

Caused by:
  failed to load source for dependency `tauri-plugin-window-state`

Caused by:
  Unable to update registry `crates-io`

Caused by:
  failed to update replaced source registry `crates-io`

Caused by:
  failed to fetch `https://rsproxy.cn/crates.io-index`

Caused by:
  process didn't exit successfully: `git fetch --force --update-head-ok https://rsproxy.cn/crates.io-index +HEAD:refs/remotes/origin/HEAD` (exit code: 128)
       Error failed to build app: failed to build app
file:///C:/Users/DELL/AppData/Roaming/npm/node_modules/pake-cli/dist/cli.js:376
                reject(new Error(`${code}`));
                       ^

Error: 1
    at file:///C:/Users/DELL/AppData/Roaming/npm/node_modules/pake-cli/dist/cli.js:376:24
    at C:\Users\DELL\AppData\Roaming\npm\node_modules\pake-cli\node_modules\shelljs\src\exec.js:145:9
    at ChildProcess.exithandler (node:child_process:427:5)
    at ChildProcess.emit (node:events:513:28)
    at maybeClose (node:internal/child_process:1091:16)
    at ChildProcess._handle.onexit (node:internal/child_process:302:5)

Node.js v18.15.0

**Comment by tw93** [comment, marked as answer]
科学上网即可，你的命令行网络环境看起来 github、crates-io 访问都有些问题，可以试试 clashx pro 的 Enhanced Mode 模式
